demo/admin/package.json

```diff
@@ -12,7 +12,7 @@
         "intl:compile": "run-p intl:compile:comet intl:compile:comet-demo",
         "intl:compile:comet": "formatjs compile-folder --format simple --ast lang/comet-lang lang-compiled/comet-lang",
         "intl:compile:comet-demo": "formatjs compile-folder --format simple --ast lang/comet-demo-lang/admin lang-compiled/comet-demo-lang-admin",
-        "intl:extract": "formatjs extract \"src/**/*.ts*\" --ignore ./**.d.ts --out-file lang-extracted/en.json --format simple",
+        "intl:extract": "formatjs extract \"src/**/*.{ts,tsx}\" --ignore ./**.d.ts --out-file lang-extracted/en.json --format simple",
```

Comment:
Why is this change necessary?

Reply:
@kaufmo fixed this in this branch during my holiday. in following commit: https://github.com/vivid-planet/comet/pull/3445/commits/dc7e2da1f2383aa8ec58ae4b0a36c9fb12c28ee0

I think this was needed because lint pipeline failed an 

> ts* also matches tsconfig.build.tsbuildinfo files which should be ignored.

Reply:
Probably not needed anymore now that it isn't part for the CLI-package.